### PR TITLE
[WFLY-12496] Use correct address for node1 in integration tests

### PR DESCRIPTION
**Pull request by istraka**
https://issues.jboss.org/browse/WFLY-12496

### CODE REVIEW

**Comment by bstansberry** [comment]
retest this please

This requires approval from @ehsavoie as he's dealing with blocker issues involving these tests and we don't want to conflict.

**Comment by bstansberry** [comment]
I put the 18.x label on this because if it passes CI and doesn't interfere with other work there's no reason not to merge it for 18.

**Comment by bstansberry** [comment]
@istraka The 'Linux with security manager' CI job failures are due to this change. Deployment needs permissions.

**Review by ehsavoie — APPROVED** [review]

File: testsuite/integration/multinode/src/test/java/org/jboss/as/test/multinode/transaction/nooutbound/TransactionContextRemoteCallTestCase.java [hunk 1]
```diff
@@ -85,15 +81,15 @@
     @Test
     @OperateOnDeployment(CLIENT_DEPLOYMENT)
     public void remotePlusHttpMandatoryBean() {
-        ProviderUrlData providerUrl = new ProviderUrlData("remote+http", url.getHost(), serverPort);
+        ProviderUrlData providerUrl = new ProviderUrlData("remote+http", TestSuiteEnvironment.getServerAddressNode1(), serverPort);
```
Comment (bstansberry) [inline testsuite/integration/multinode/src/test/java/org/jboss/as/test/multinode/transaction/nooutbound/TransactionContextRemoteCallTestCase.java]: This test now reads a system property so I think the set of permissions added to CLIENT_DEPLOYMENT in clientDeployment() needs to include a PropertyPermission.

**Review by bstansberry — CHANGES_REQUESTED** [review]

**Comment by kabir** [comment]
@istraka if you agree can you add the missing permission? Or if not, leave a message

**Comment by kabir** [comment]
Superseded by 12779

**Pull request closed without merging**